### ISSUE TITLE
Save Contraptions to the Cloud (or something)

### ISSUE THREAD

**Issue by Roozz1** [studio-minus/ppg-bugs-and-requests#1208]
**Feature description**
Save non-workshop contraptions to the cloud, that way if someone reinstalls the game, their contraptions will still exist.

**Feature purpose**
I accidentally deleted the game from my SSD so I reinstalled it, and poof, all my contraptions are gone. It would be extremely extremely extremely handy if contraptions were atleast backed up somewhere locally or backed up to the Steam cloud.

This is probably a feature already but if it isn't, you should really add this lol.

**Additional context**
n/a

**Comment by DindinYT37** [comment]
Makes sense but you could also just upload and subscribe to them.

**Comment by Roozz1** [comment]
true, although it would be pretty nice if it didnt require you to upload ur creations to the workshop or whatever

**Comment by mestiez** [comment]
I can enable Steam Cloud saving, but this has introduced issues in the past.
